sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -379,10 +379,54 @@ export interface KeyValuePairModel {
 }
 
 // @public
-export type KeyValueType = "string" | "selectionMark" | string;
+export type KeyValueType = string;
 
 // @public
-export type Language = "en" | "es" | "de" | "fr" | "it" | "nl" | "pt" | "zh-Hans" | string;
+export const enum KnownKeyValueType {
+    // (undocumented)
```

Comment:
This (undocumented) is part of the generated code that comes from the swagger. And Adding documentation for enum values does not make much sense. So, I think this is acceptable.